docs/source/release_notes.rst

```diff
@@ -7,14 +7,20 @@ Future Release
 ===============
     * Enhancements
         * Add ``'passthrough'`` and ``'ignore'`` to tags in ``list_semantic_tags`` (:pr:`1094`)
+        * Add initialize with partial table schema  (:pr:`1100`)
         * Apply ordering specified by the ``Ordinal`` logical type to underlying series (:pr:`1097`)
     * Fixes
     * Changes
     * Documentation Changes
     * Testing Changes
 
     Thanks to the following people for contributing to this release:
-    :user:`davesque`, :user:`tamargrey`
+    :user:`davesque`, :user:`tamargrey`, :user:`tuethan1999`
+    
+Breaking Changes
+++++++++++++++++
+    * Calling `init` with a partial schema now calls `init_with_partial_schema`
```

Comment:
@tuethan1999 Can you expand on this? We need to explain the behavior before the change, and the new behavior now. See [here](https://woodwork.alteryx.com/en/latest/release_notes.html#breaking-changes) for an example